vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/SpreadsheetTable.java

```diff
@@ -67,10 +72,13 @@ public SpreadsheetTable(Spreadsheet spreadsheet,
      *            Cell range to build the table in
      */
     public SpreadsheetTable(Spreadsheet spreadsheet, Sheet sheet,
-            CellRangeAddress fullTableRegion) {
+            CellRangeAddress fullTableRegion,
+            CTAutoFilter ctWorksheetAutoFilter, XSSFTable xssfTable) {
```

Comment:
Changing this constructor without maintaining the old one would cause breaking changes if anyone else uses it. The class is public, so we should be careful with this type of change. In general, ideally, we should try not to add any new API as part of a fix.

Reply:
Yes, you are correct. I've re-added the constructor in a new commit.
I understand that adding a new API as part of the fix is not nice, however, the other options on how to fix this issue were even worse in my opinion. Because initPopupButtons() is called in the constructor, I need to set the `xssfTable`/`ctWorksheetAutoFilter` before this call - and that is doable only via a new constructor.